include/uv/win.h

```diff
@@ -62,6 +62,9 @@ typedef struct pollfd {
 #include "uv/tree.h"
 #include "uv/threadpool.h"
 
+/* MAX_PATH is in characters, not bytes. Make sure we have enough headroom. */
+#define UV_PATH_MAX_BYTES (MAX_PATH * 4)
```

Comment:
The maximum is `(32k * 4)` bytes (4x char count to fit ucs-2 expansion into utf-8)

(the last API hold-out that was still using `MAX_PATH` was getcwd, and that's been configurable now in the registry with the LongPathsEnabled setting for a couple years now)